include/Smilies.php

```diff
@@ -0,0 +1,182 @@
+<?php
+
+/**
+ * @file include/Smilies.php
+ * @brief This file contains the Smilies class which contains functions to handle smiles
+ */
+
+/**
+ * This class contains functions to handle smiles
```

Comment:
they should work because they use the `smilie` hook. I have tested smiley-button, smiley pack and adult smilies. Are there any other addons which should be tested?